### DISCUSSION TITLE
Your application has a total of 1 package(s) that you may not be legally allowed to use. By default, we assume the MIT, Apache-2.0, ISC, BSD Clause 2 & 3 and LGPL licenses to be legally valid for use for proprietary or commercial applications. However, you are free to change this in the Enlightn config. Unsafe packages include [mpdf/mpdf: GPL-2.0-only]

### DISCUSSION THREAD

**Discussion by shakil-brl** [mpdf/mpdf#1718] [Q&A]
Your application has a total of 1 package(s) that you may not be legally allowed to use. By default, we assume the MIT, Apache-2.0, ISC, BSD Clause 2 & 3 and LGPL licenses to be legally valid for use for proprietary or commercial applications. However, you are free to change this in the Enlightn config. Unsafe packages include [mpdf/mpdf: GPL-2.0-only]

What can I do ?

**Comment by ssieb** [comment]
What are you using that tells you GPL is not legal for for commercial applications?
